agents/rebase_agent.py

```diff
@@ -95,12 +96,13 @@ def get_instructions() -> str:
       8. In your output, provide a "files_to_git_add" list containing all files that should be git added for this rebase.
          This typically includes the updated spec file and any new/modified/deleted patch files or other files you've changed
          or added/removed during the rebase. Do not include files that were automatically generated or downloaded by spectool.
+         Make sure to include patch files that were also removed from the spec file.
 
 
       General instructions:
 
       - If necessary, you can run `git checkout -- <FILE>` to revert any changes done to <FILE>.
-      - Never change anything in the spec file changelog.
+      - Never change anything in the spec file changelog. Changelog entry will be taken care of by the log agent.
```

Comment:
I hate that it ignores this. Either way, are both changes really needed, isn't the previous one enough?